app/queries/books/find_books.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Books
+  class FindBooks
```

Comment:
Follow the naming guideline
```suggestion
  class FilterQuery
```

Reply:
And rename the file